spec/decorators/conviction_decorator_spec.rb

```diff
@@ -29,4 +29,31 @@
       it { expect(subject.custodial_sentence?).to eq(false) }
     end
   end
+
+  describe '#motoring?' do
```

Comment:
Then we don't need these tests (as these will be tested already via proxy the decision tree).